### DISCUSSION TITLE
Is there any tool in Capacitor that is working the same way as `Camera.getPhoto` - but for videos?

### DISCUSSION THREAD

**Discussion by alexeysarakuz** [ionic-team/capacitor#7591] [Q&A]
In my application I need to allow users to record videos with native camera OR get video from gallery, the same way it works for `Camera.getPhoto` + `CameraSource.Prompt` option

**Comment by robingenz** [comment]
You can use the [Capacitor File Picker](https://capawesome.io/plugins/file-picker/) plugin to let the user select a video from the gallery. To record a video, I recommend the [Capacitor Camera Preview](https://github.com/capacitor-community/camera-preview) plugin.

**Comment by jcesarmobile** [comment]
You can use a html input of type file with accept set to video like this: `<input type="file" accept="video/*" />`
